Bottom: cd9d9090ca20d26e19a52540a68ae1260c248951
Top:    e4a16be321a3c1b668844fb3eb86a3f7936e44b0
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-02-18 18:04:50 +0000

polygon.h: Reduce circ_segs to 8 for debug clarity


---

diff --git a/src/polygon.h b/src/polygon.h
index 369efea..0643765 100644
--- a/src/polygon.h
+++ b/src/polygon.h
@@ -34,7 +34,7 @@
 
 /* Implementation constants */
 
-#define POLY_CIRC_SEGS 40
+#define POLY_CIRC_SEGS 8 //40
 #define POLY_CIRC_SEGS_F ((float)POLY_CIRC_SEGS)
 
 #if 0
